codalab/lib/emailer.py

```diff
@@ -33,6 +33,7 @@ def __init__(self, host, user, password, default_sender, server_email, port=587,
         self.server_email = server_email
         self.port = port
         self.use_tls = use_tls
+        self.do_login = self.password != None
```

Comment:
Add parentheses around the `self.password != None` part of the expression for clarity.